docker/pravega-schema-registry/Dockerfile-java-11

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+FROM openjdk:11.0-jre-slim
+
+RUN apt-get update && apt-get install -y -q \
+    rpcbind \
+    nfs-common \
+    python \
```

Comment:
Do we require these packages?

Reply:
i have trimmed it to remove nfs-common and nfs-util. 
i dont think we need rpcbind or gcompat or for that matter any of these.